src/main/java/ch/wisv/areafiftylan/security/authentication/PasswordChangeDTO.java

```diff
@@ -29,7 +31,7 @@
     String oldPassword = "";
     @Getter
     @Setter
-    @NotEmpty
+    @Length(min = 6)
```

Comment:
Can't we use a ✨magic✨ number here, since somebody might forget that not only the resetPassword and changePassword have to be changed, too?